### PR TITLE
diff replace derivative in docs

### PR DESCRIPTION
**Pull request by jtlap**
only html docs are modified here

### CODE REVIEW

File: docs/reference/functions/arithmetic.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
 [`eve::negatenz`](arithmetic/negatenz.html)
 :   computes the product of its parameter by the sign of the second one while considering the sign of 0 to be one
 [`eve::oneminus`](arithmetic/oneminus.html)
-:   computes the difference between 1 and its parameter without lose of precision ($1 - x$)
+:   computes the derivativeerence between 1 and its parameter without lose of precision ($1 - x$)
```
Comment (jfalcou) [inline docs/reference/functions/arithmetic.html:71]: this diff->derivative looks bad

File: docs/reference/functions/arithmetic/average.html [hunk 2]
```diff
@@ -50,7 +50,7 @@
 For two parameters half the sum of `x` and `y`. No overflow occurs.
 
 For more than two parameters only floating entries are allowed and overflow is  generally avoided using Knuth algorithm.
-However, problems can arise if the difference of two parameters
+However, problems can arise if the derivativeerence of two parameters
```
Comment (jfalcou) [inline docs/reference/functions/arithmetic/average.html:53]: Same error

File: docs/reference/functions/operators/div.html [hunk 3]
```diff
@@ -45,7 +45,7 @@
 Return value
 ----------------------------------------------------------------------------------------------------
 <span class="smallskip"></span>
-Return the difference of `x` and `y`.
+Return the derivativeerence of `x` and `y`.
```
Comment (jfalcou) [inline docs/reference/functions/operators/div.html:48]: Again

File: docs/reference/functions/operators/sub.html [hunk 4]
```diff
@@ -45,7 +45,7 @@
 Return value
 ----------------------------------------------------------------------------------------------------
 <span class="smallskip"></span>
-Return the difference of `x` and `y`.
+Return the derivativeerence of `x` and `y`.
```
Comment (jfalcou) [inline docs/reference/functions/operators/sub.html:48]: Same here

File: docs/reference/functions/operators/sub.html [hunk 5]
```diff
@@ -64,19 +64,19 @@
...
+    diffs of $f$, where $f$ is the function $(x,y) \rightarrow \ x-y$.
 
 Masked calls
 ====================================================================================================
  The [masked](../../../tutorial/masking.html) calls `sub[cond](x,y)` and `saturated(sub[cond])(x,y)` where `cond` satisfies the
  [conditionnal-expression](../../concepts.html#conditionnal) requirements are supported.
 
  More precisely, if `cond` is
- a mere [logical value](../../concepts.html#value) these expressions compute the difference (resp the saturated
+ a mere [logical value](../../concepts.html#value) these expressions compute the derivativeerence (resp the saturated
```
Comment (jfalcou) [inline docs/reference/functions/operators/sub.html:79]: Again

File: docs/tutorial/masking.html [hunk 6]
```diff
@@ -55,7 +55,7 @@
 The [if_else][doc_if_else] call explicitly requests we pass:
  - the condition, i.e the comparison between `a` and `b`
  - the value to use whenever an element of said condition evaluates to `true`, here the product of `a` and `b`
- - the value to use whenever an element of said condition evaluates to `false`, here the difference of `a` and `b`
+ - the value to use whenever an element of said condition evaluates to `false`, here the derivativeerence of `a` and `b`
```
Comment (jfalcou) [inline docs/tutorial/masking.html:58]: Again again

File: docs/tutorial/masking.html [hunk 7]
```diff
@@ -255,7 +255,7 @@
 One may want for example to not compute an expression on the first and last element of such
 [SIMD values](reference/concepts.html#simd_value).
 
-Let's write a function that computes the differences between two vectors in multiple scenarios: not
+Let's write a function that computes the derivativeerences between two vectors in multiple scenarios: not
```
Comment (jfalcou) [inline docs/tutorial/masking.html:258]: and again :)

**Review by jfalcou — CHANGES_REQUESTED** [review]
you have been a bit too strong with your searcg//repalce

File: docs/reference/functions/arithmetic.html (hunk 1 above)
Comment (jtlap) [reply docs/reference/functions/arithmetic.html]: done

File: docs/reference/functions/arithmetic/average.html (hunk 2 above)
Comment (jtlap) [reply docs/reference/functions/arithmetic/average.html]: done

File: docs/reference/functions/operators/div.html (hunk 3 above)
Comment (jtlap) [reply docs/reference/functions/operators/div.html]: done

File: docs/reference/functions/operators/sub.html (hunk 4 above)
Comment (jtlap) [reply docs/reference/functions/operators/sub.html]: done

File: docs/reference/functions/operators/sub.html (hunk 5 above)
Comment (jtlap) [reply docs/reference/functions/operators/sub.html]: done

**Review by jfalcou — APPROVED** [review]

**Pull request merged**